test/pynrn/test_units.py

```diff
@@ -0,0 +1,68 @@
+from neuron import h
+
+def switch_units(legacy):
+  try:
+    h.nrnunit_use_legacy(legacy)
```

Comment:
@nrnhines : is this documented somewhere in user docuemtnaiton now?

Reply:
Yes. https://neuronsimulator.github.io/nrn/py_doc/simctrl/programmatic.html#nrnunit_use_legacy